upup/pkg/fi/cloudup/openstacktasks/servergroup.go

```diff
@@ -62,9 +62,11 @@ func (s *ServerGroup) Find(context *fi.Context) (*ServerGroup, error) {
 				return nil, fmt.Errorf("Found multiple server groups with name %s", fi.StringValue(s.Name))
 			}
 			actual = &ServerGroup{
-				Name:    fi.String(serverGroup.Name),
-				ID:      fi.String(serverGroup.ID),
-				Members: serverGroup.Members,
+				Name:      fi.String(serverGroup.Name),
+				ID:        fi.String(serverGroup.ID),
+				Members:   nil, // TODO implement logic how servergrp members are updated
```

Comment:
I can open a PR for the appropriate way of updating if desirable.  Okay with this stopgap for now.

Reply:
This is consumed by some downstream tasks, but not past iterating through the members.  This should be a safe operation.